notebooks/text-generation/mistral-inference-tgi-sagemaker.ipynb

```diff
@@ -0,0 +1,1041 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3ed8bd00-eda6-4fd5-9910-da9c671e5acf",
+   "metadata": {},
+   "source": [
+    "# Deploy Mistral to SageMaker + Inferentia2 using HF Optimum Neuron and TGI\n",
+    "\n",
+    "This guide will detail how to compile, deploy and run a **Mistral 7B** on AWS inferentia 2 + Amazon SageMaker.\n",
+    "\n",
+    "You will learn how to:\n",
+    "- set up your AWS instance,\n",
+    "- compile Mistral-7B model to the Neuron format using a SageMaker Training Job,\n",
+    "- deploy the model and use it in any application.\n",
+    "\n",
+    "Note: This tutorial was created on Amazon SageMaker.\n",
+    "\n",
+    "## Prerequisite: Setup AWS environment\n",
+    "\n",
+    "*you can skip that section if you are already this notebook on SageMaker Studio.*\n",
```

Comment:
Missing word ? "if you are already running this notebook" ?

Reply:
fixed. tnx.